website/docs/kubernetes/configuring-editing-kube-object.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 12
+title: Managing objects
+description: Creating and updating a kubernetes object
+keywords: [podman desktop, podman, deploying, objects, kubernetes]
+tags: [managing-kubernetes, objects]
+---
+
+# Managing Kubernetes objects
+
+With Podman Desktop, you can enhance your Kubernetes management experience by using the features that are part of the Kubernetes explorer. As a developer, you can easily transition from containers to Kubernetes and deploy a complete Kubernetes environment with necessary objects. You have the ability:
+
+- To control and manage your application resources visually
+- To configure the following Kubernetes objects:
+  - `Node`: Use this object to set up a node on which the necessary pods can run within a kubernetes cluster.
+  - `Deplyoment`: Use this object to create necessary pods for execution and scale the number of pods.
+  - `Service`: Use this object to expose your application to users and define policies for application access.
+  - `Ingress`: Use this object to define routing rules and manage user access to the services running in a Kubernetes cluster.
+  - `PersistentVolumeClaim`: Use this object to request `PersistentVolume` resources for storage and define volume access modes within your Kubernetes cluster.
+  - `ConfigMap`: Use this object to to define non-sensitive configuration data for initializing or executing your application.
+  - `Secret`: Use this object to store and manage sensitive data, such as passwords, OAuth tokens, and SSH keys for your application.
+- To view and analyze real-time information about the connection status of the resources configured within the cluster
+- To get resource details using the _Summary_ and _Inspect_ tabs
+- To edit and apply configuration changes directly using the _Kube_ tab
+- To select multiple configuration files and apply them to your cluster in a single step
```

Comment:
Every sentence repeats "To" at the beginning.

Update the sentence: You have the ability: to `You have the ability to:` and remove every "To" in this list?